lang/jvm/src/test/scala/com/wavesplatform/lang/EvaluatorV1Test.scala

```diff
@@ -377,4 +378,30 @@ class EvaluatorV1Test extends PropSpec with PropertyChecks with Matchers with Sc
       )
     )
   }
+
+  property("checking a hash of some message by crypto function invoking") {
+    val bodyText      = "some text for test"
+    val bodyBytes     = bodyText.getBytes()
+    val hashFunctions = Map("sha256" -> Sha256, "blake2b256" -> Blake2b256, "keccak256" -> Keccak256)
+
+    for ((funcName, funcClass) <- hashFunctions) hashFuncTest(bodyBytes, funcName) shouldBe Right(ByteVector(funcClass.hash(bodyText)))
+  }
+
+  private def hashFuncTest(bodyBytes: Array[Byte], funcName: String): Either[(Context, ExecutionLog, ExecutionError), ByteVector] = {
+    val context = Monoid.combineAll(
+      Seq(
+        PureContext.instance,
+        CryptoContext.build(Global)
```

Comment:
meh. global is not a thing we use in node, it's WavesCrypto.instance in main package